testing_and_setup/compass/ocean/ziso/ziso_run.xml

```diff
@@ -4,8 +4,6 @@
 		<option name="config_ocean_run_mode">'forward'</option>
 		<option name="config_use_mom_del4">.true.</option>
 		<option name="config_mom_del4_div_factor">10.0</option>
-		<option name="config_convective_visc">1.0</option>
-		<option name="config_convective_diff">1.0</option>
```

Comment:
Same thing-- no addition here?  I'm a little confused and am missing something.

Reply:
These are already the default values (both with and without CVMix):
https://github.com/MPAS-Dev/MPAS-Model/blob/ocean/develop/src/core_ocean/Registry.xml#L471-L478
They could be added explicitly, though, if we think it's meaningful.  I personally don't.

Reply:
Agreed